src/ProjectCracker/ProjectCracker.fs

```diff
@@ -76,12 +82,14 @@ type JsonValue with
         result
 
 let private frameworkPreference = [
+    "netcoreapp3.0", ".NETCoreApp,Version=v3.0";
```

Comment:
Could you add netcoreapp3.1 if it is possible.

Reply:
Sure, I included it in https://github.com/fsprojects/fsharp-language-server/pull/70/commits/900bc8dc003ee64e968a90b1c457f96b59f8513b.

Reply:
Thank you!